website/docs/execution-node/fee-recipient.md

```diff
@@ -291,6 +203,24 @@ See [this issue](https://github.com/prysmaticlabs/prysm/issues/11322) to track t
 
 :::
 
+
+## Frequently asked questions
+
+**How do I know if fee recipient was properly configured?** <br />
+If you don't see any errors after issuing one of the above commands, your fee recipient address has been successfully configured.
+
+**What happened to `fee-recipient-config-file`?** <br />
+`fee-recipient-config-file` and `fee-recipient-config-url` flags are deprecated and have been replaced with `proposer-settings-file` and `proposer-settings-url` flags as of Prysm v2.1.3.
+
+**TODO: what does this mean?** <br />
+- The existing `suggested-fee-recipient` flag must now be used with the new `enable-builder` flag. This allows your validator client to use the MEV Builder API.
+
+**TODO: when should I set my own `gas_limit`, and how do I know what to set?** <br />
```

Comment:
this will notify the relay the gas limit you would like the network to strive towards, there is a limited amount the network can move the gas limit per block so I think it's more something that is focused on at the social layer. pretty advanced and probably won't need to change until the community as a whole wants to shift in some way but builders need to have this power. [
](https://ethereum.org/en/developers/docs/gas/)

Reply:
@james-prysm hmmm... how do we think users should think about this, in nontechnical terms? What happens (to me, my rewards, the network, etc) if I increase the `gas_limit`? What happens if I decrease it? When should I change it? Trying to explain it using "minimum-viable technical complexity" 🤔

Reply:
https://github.com/ethereum/builder-specs/issues/17
I asked the same thing , here's a summary if gas_limits get to an undesirable outcome. 
> Sending something very large will result in you not being able to include that many transactions in a block and sending too little the block is not profitable. Will those be adjusted somehow or is it totally up to the validator client.

so validators sort of vote on what the target should be to maximize this...